src/client/components/QuizAnswers/QuizAnswers.styles.css

```diff
@@ -0,0 +1,77 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@100;200;400;700&display=swap');
```

Comment:
I think you can remove this, as there's going to be a PR to handle importing fonts

Reply:
Thank you, Beth, I will.

Reply:
Yes, that PR is ready now. So the import can be removed @sofiiadidovych :)